src/stitches.config.ts

```diff
@@ -85,6 +85,7 @@ export const colors = {
   surface: figmaColors.grey2,
   border: figmaColors.grey4,
   focusedBorder: figmaColors.grey8,
+  dividingLine: figmaColors.grey8,
```

Comment:
bikeshedding a bit here but maybe we should rename `focusedBorder` to `heavyBorder` and use that here instead of creating a new name

Reply:
The usage of "focusedBorder" is pretty all over the place. Nav link active underlines, external link icon color, users list on map scroll bar thumb, checkbox border, and then "focused text area border" , the only place where the name lives up. I think this is alright to rename but I don't love the kind of fake semantics here.

Reply:
I'm actually going to use `borderMedium` since this is room for a darker one and there is a patten here with `secondary` and `secondaryDark`